### PR TITLE
adds emacs docs to quit and open file

### PR DESCRIPTION
**Pull request by sethwoodworth**

### CODE REVIEW

**Comment by waldyrious** [comment]
Isn't it a little awkward to start with the command to exit emacs? I'd swap the order of the examples. Other than that, LGTM.

**Comment by rubenvereecken** [comment]
I'd second @waldyrious

File: pages/common/emacs.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# emacs
+
+> A command-line text editor (and operating system)
+
+- Exit emacs
+
+`CTRL`+`x`, `CTRL`+`c`
```
Comment (ostera) [inline pages/common/emacs.md]: I think the notation for this is `C-x C-c`

**Comment by sethwoodworth** [comment]
@waldyrious As a non-emacs user, I've found myself in more situations where I've been trying to quit emacs than start it.  Happy to sort it by 'sane' order rather than my pass at most-frequently asked.
@ostera Will-do

**Comment by sethwoodworth** [comment]
@waldyrious Reordered, and merged with #491

File: pages/common/emacs.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/emacs.md]: Here the 2/3 LGTM policy would have helped; I would argue that this notation is an emacs idiom and we should probably use something a little more generic, aimed at beginners. You made this point yourself recently in another PR regarding `cr` vs. `enter`.

**Pull request merged**